cluster-provision/k8s/1.18/provision.sh

```diff
@@ -272,7 +271,7 @@ etcd:
     dataDir: /var/lib/etcd
 imageRepository: k8s.gcr.io
 kind: ClusterConfiguration
-kubernetesVersion: ${version}
+kubernetesVersion: v${version}
```

Comment:
Remove 'v' :D

Reply:
nope, this is how it should be in automatic migrate, also see please the original parameter that was given with 'v'

Reply:
Ohh ok got it now